### PR TITLE
chore: add 0.12.0 release notes

### PR DESCRIPTION
**Pull request by rokroskar**
Release notes for renku version `0.12.0`.

### CODE REVIEW

**Review by jachro — COMMENTED** [review]
👍

File: CHANGELOG.rst
Comment (ciyer) [inline CHANGELOG.rst:38]: In addition to what is already mentioned above, we have the following on deck for our changelog for the release:

```
* improved display of math formulas in markdown
* more robust autosave infrastructure
* support for project-template-defined parameter validation
* support for icons and description for project templates
* (bug fix) fix problems with dataset "add to project" button
```

The full notes are here: https://github.com/SwissDataScienceCenter/renku-ui/releases

**Review by ciyer — CHANGES_REQUESTED** [review]
This looks good. There are a few more things from the UI side that might be worth mentioning, but it is your call on whether they make sense in the release nodes.

**Review by Panaetius — DISMISSED** [review]

**Review by jachro — APPROVED** [review]
Thanks for putting that together

File: CHANGELOG.rst
Comment (ableuler) [inline CHANGELOG.rst:57]:
```suggestion
earlier iterations, but depending on the number of projects it could still take
```

Comment (rokroskar) [reply CHANGELOG.rst:57]: 🧐

**Review by ableuler — COMMENTED** [review]
@olevski now all the pieces for the custom CA certificates are included in the upcoming release, right?

File: CHANGELOG.rst
Comment (olevski) [inline CHANGELOG.rst:49]:
```suggestion
* Notebooks: https://github.com/SwissDataScienceCenter/renku-notebooks/releases

```

There are mostly bug fixes on the notebook side. And the s3 buckets feature. But the S3 buckets feature is off by default and is not yet supported by the UI. So I do not want to advertise that yet. In the release when the UI fully supports this we can talk about the s3 storage stuff. Same goes with the custom certificates - not all components have releases that include that. So it is better to talk about that in the release when all the components are ready.

**Comment by olevski** [comment]
> @olevski now all the pieces for the custom CA certificates are included in the upcoming release, right?

No the gateway is still not tagged for a new release because of the issues with its helm chart (which we have addressed with a PR). 

Also some components have relases here that went out before the custom certificates stuff was merged.

File: CHANGELOG.rst
Comment (olevski) [reply CHANGELOG.rst:50]: I guess the alternative to not saying anything we can mention that there is partial support for s3 bucket storage and for custom CA certificates.

File: CHANGELOG.rst
Comment (olevski) [inline CHANGELOG.rst:42]:
```suggestion
* (bug fix) fix problems with dataset "add to project" button
* partial support for custom CA certificates
* experimental support for s3 storage in user sessions
```

**Comment by olevski** [comment]
Looks good to me!

**Review by olevski — APPROVED** [review]

**Review by ableuler — DISMISSED** [review]

**Comment by rokroskar** [comment]
cc @SwissDataScienceCenter/ordes please have a look ☝️

File: CHANGELOG.rst
Comment (ciyer) [inline CHANGELOG.rst:44]: We decided to call this "cloud storage" and should be consistent with the wording.
```suggestion
* experimental support for cloud storage in user sessions
```

**Review by ciyer — CHANGES_REQUESTED** [review]
I think we should be consistent in talking about "cloud storage;"  S3 is a bit confusing since it could me AWS S3, it could me the S3 API, which does not have to be hosted on AWS.

**Review by Panaetius — APPROVED** [review]

**Review by cmdoret — APPROVED** [review]
:+1:

**Pull request merged**
